docs/schema.md

```diff
@@ -607,6 +614,66 @@ The known reference `type` values are:
 - `PACKAGE`: A web page for the affected package itself.
 - `WEB`: A web page of some unspecified kind.
 
+## credits fields
+
+```json
+{
+	"credits": [ {
+		"name": string,
+		"contact": [ {
+			"type": string,
+			"value": string
+		} ],
+	} ]
+}
+```
+
+The `credits` field is a JSON array providing a way to give credit where credit
```

Comment:
nit: just "give credit for the discovery, confirmation, patch..." is more concise?